llvm/lib/Target/SPIRV/SPIRVBuiltins.h

```diff
@@ -38,6 +38,12 @@ std::optional<bool> lowerBuiltin(const StringRef DemangledCall,
                                  const SmallVectorImpl<Register> &Args,
                                  SPIRVGlobalRegistry *GR);
 
+/// Helper external function for finding a builtin funtion attributes
```

Comment:
Typo: funtion -> function

Why do you say this function is external? I think all functions declared in SPIRVBuiltins.h are defined in SPIRVBuiltins.cpp

Reply:
Thank you, I fix this

Reply:
@michalpaszkowski done

Reply:
Thank you!